src/Stepper/StepButton.js

```diff
@@ -0,0 +1,147 @@
+import React, {PropTypes} from 'react';
+import transitions from '../styles/transitions';
+import EnhancedButton from '../internal/EnhancedButton';
+import StepLabel from './StepLabel';
+
+const isLabel = (child) => {
+  return child && child.type && child.type.muiName === 'StepLabel';
+};
+
+const getStyles = (props, context, state) => {
+  const {hovered} = state;
+  const {backgroundColor, hoverBackgroundColor} = context.muiTheme.stepper;
+
+  const styles = {
+    root: {
+      padding: 0,
+      backgroundColor: hovered ? hoverBackgroundColor : backgroundColor,
+      transition: transitions.easeOut(),
+    },
+  };
+
+  if (context.stepper.orientation === 'vertical') {
+    styles.root.width = '100%';
+  }
+
+  return styles;
+};
+
+class StepButton extends React.Component {
```

Comment:
Did I extend Component in another by accident?

Sorry, that's because of my own snippets :smile: I'll remove for now to be consistent with the rest of the codebase.